plugins/modules/gns3_project.py

```diff
@@ -51,6 +51,18 @@
         description:
             - Project name
         type: str
+    project_path:
+        description:
+            - Project path (Ignore in name not specified)
```

Comment:
Shouldn't it be "Ignore if name not specified"?

Reply:
Or better yet, something like "(Optional) Path of the project on the server"

Reply:
How about :  Path of the project on the server ( Optional,  Ignored if project_name not specified )